test/helpers.js

```diff
@@ -111,22 +111,29 @@ export async function getSchemeVotingMachine(dao, scheme, ndxVotingMachineParame
 }
 
 export async function getVotingMachineParameters(votingMachine, votingMachineParamsHash) {
-  return votingMachine.parameters(votingMachineParamsHash);
+  return votingMachine.contract.parameters(votingMachineParamsHash);
 }
 
 /**
  * vote for the proposal given by proposalId.
- * votingMachine must be the raw contract, not a wrapper.
  */
 export async function vote(votingMachine, proposalId, vote, voter) {
   voter = (voter ? voter : accounts[0]);
-  votingMachine = votingMachine.contract ? votingMachine.contract : votingMachine;
-  return await votingMachine.vote(proposalId, vote, { from: voter });
+  if (votingMachine.contract) {
```

Comment:
I dot really understand this conditional, can you comment?

Reply:
added a comment in the code